keps/sig-windows/2578-windows-conformance/README.md

```diff
@@ -0,0 +1,418 @@
+- Ability to delete and recreate pods for StatefulSets which preserve their ability to serve as routed endpoints for services, while also having unchanging IP addresses.
+- Ability to access internal pods by internal Kubernetes DNS service endpoints.
+- Ability to access external services by Kubernetes DNS services endpoints (for non-airgapped clusters).
+- Ability to access internal pods through custom DNS records, injected by the Kubernetes pod Specification.
+- Ability to route services from pods from the EndpointSlice API.
+
+#### Basic Service accounts and local storage
+
+- Ability to rotate service accounts for running pods, with the APIServer access capacity of the pod remaining unchanged over the long term.
+- Ability to read and write shared files on a single Kubernetes node between three running Windows containers, simultaneously.
+
+#### Basic Scheduling
+
+- Ability to schedule pods with CPU and Memory limits demonstrably honored over time.
+- Ability to demonstrate the pods requesting more CPU and Memory then available are left in the Pending state over time.
+
+#### Basic performance
+
+- Ability to reschedule the a deployment Pod, 50 times in a row, with continuous deletions in the background.
+- Ability to route traffic to 10 pods behind a common ClusterIP service endpoint.
+- Ability to route traffic to 10 pods behind a common NodePort service endpoint.
+
+We do not include GMSA support in the basic definition of WindowsConformance, because it is known to require other cluster components outside of Kubernete's Control.
+
+### Sub-specifications of Windows Conformance (Kubernetes 1.23)
+
+The following subsets of verifications for Windows clusters further expand the way we define supportability for Windows clusters, but they are not considered part of the "core" Conformance specification.  These features may be *heavily* reliant on functionality which resides well outside of Kubernete's preview, for example: CNI Implementation, Container Runtime implementation, or Windows Server Edition + Configuration.
+
+#### Windows HostProcess Conformance
+
+Note that, HostProcess Containers have not yet been verified to support all of these features.
+Nevertheless, we define this specification because it is an obvious starting point for the long-term, meaningfull support of a full-fledged HostProcess feature in Kubernetes.
+
+- Ability to access the APIServer using pod mounted service accounts from a hostProcess pod.
+- Ability to create and manage host level networking (hcn) rules from a Windows hostProcess pod.
+- Ability to launch hostProcess containers which share the IP address of a Windows node.
+- Ability to launch hostProcess containers which can run other privileged Windows system API (to be specified further in the future).
+- Ability to utilize NT Authority/Network Service should be able to interact with host network objects
```

Comment:
I don't dont think this actually works today, it seems to need NT Authority/System?

Reply:
ack, ok removed